lnprototest/backend/bitcoind.py

```diff
@@ -71,16 +72,30 @@ def __init__(self, basedir: str):
             "-logtimestamps",
             "-nolisten",
         ]
-        self.port = reserve()
+        self.port = None
```

Comment:
This "initialize to None" is a bad habit.  You should just not set them at all if you don't expect them to be accessed.